rush.json

```diff
@@ -1,483 +0,0 @@
-   * CI validation to build the entire repo twice: once with the old release, and once with the new release.
-   *
-   * Rush "installation variants" correspond to sets of config files located under this folder:
-   *
-   *   common/config/rush/variants/<variant_name>
-   *
-   * The variant folder can contain an alternate common-versions.json file.  Its "preferredVersions" field can be used
-   * to select older versions of dependencies (within a loose SemVer range specified in your package.json files).
-   * To install a variant, run "rush install --variant <variant_name>".
-   *
-   * For more details and instructions, see this article:  https://rushjs.io/pages/advanced/installation_variants/
-   */
-  "variants": [
-    // {
-    //   /**
-    //    * The folder name for this variant.
-    //    */
-    //   "variantName": "old-sdk",
-    //
-    //   /**
-    //    * An informative description
-    //    */
-    //   "description": "Build this repo using the previous release of the SDK"
-    // }
-  ],
-  /**
-   * Rush can collect anonymous telemetry about everyday developer activity such as
-   * success/failure of installs, builds, and other operations.  You can use this to identify
-   * problems with your toolchain or Rush itself.  THIS TELEMETRY IS NOT SHARED WITH MICROSOFT.
-   * It is written into JSON files in the common/temp folder.  It's up to you to write scripts
-   * that read these JSON files and do something with them.  These scripts are typically registered
-   * in the "eventHooks" section.
-   */
-  // "telemetryEnabled": false,
-  /**
-   * Allows creation of hotfix changes. This feature is experimental so it is disabled by default.
-   */
-  // "hotfixChangeEnabled": false,
-  /**
-   * (Required) This is the inventory of projects to be managed by Rush.
-   *
-   * Rush does not automatically scan for projects using wildcards, for a few reasons:
-   * 1. Depth-first scans are expensive, particularly when tools need to repeatedly collect the list.
-   * 2. On a caching CI machine, scans can accidentally pick up files left behind from a previous build.
-   * 3. It's useful to have a centralized inventory of all projects and their important metadata.
-   */
-  "projects": [
-    {
-      /**
-       * The NPM package name of the project (must match package.json)
-       */
-      "packageName": "office-ui-fabric-react",
-      /**
-       * The path to the project folder, relative to the rush.json config file.
-       */
-      "projectFolder": "packages/office-ui-fabric-react",
-      /**
-       * An optional version policy associated with the project.  Version policies are defined
-       * in "version-policies.json" file.  See the "rush publish" documentation for more info.
-       * NOTE: "versionPolicyName" and "shouldPublish" are alternatives; you cannot specify them both.
-       */
-      "versionPolicyName": "lockedMajor"
```

Comment:
Have you implemented anything with beachball yet to prevent major bumps? Seems safest to have it enforced by tooling somewhere not just by hoping reviewers catch it.